js/main.js

```diff
@@ -69,11 +69,11 @@ function createNewsItems(dataArray) {
     var headlineImage = document.createElement('img');
     var headlineAnchor = document.createElement('a');
     if (dataArray[i]['metadata:id']) {
-      headlineText.textContent = dataArray[i].title.slice(0, 50) + ' . . .';
+      headlineText.textContent = dataArray[i].title.slice(0, 45) + ' . . .';
       headlineImage.setAttribute('src', dataArray[i]['metadata:image']['metadata:imagepath']);
       headlineAnchor.setAttribute('href', dataArray[i].link);
     } else {
-      headlineText.textContent = dataArray[i].headline.slice(0, 50) + ' . . .';
+      headlineText.textContent = dataArray[i].headline.slice(0, 45) + ' . . .';
```

Comment:
You can do this with CSS using the `text-overflow` property 🤓 

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
![Screen Shot 2021-03-03 at 11 31 23 AM](https://user-images.githubusercontent.com/75342275/109861284-ffa29500-7c13-11eb-88e6-9d39befbe8bc.png)